.eslintrc.json

```diff
@@ -9,7 +9,8 @@
         "object-shorthand": "off",
         "no-underscore-dangle": ["error", { "allowAfterThis": true }],
         "comma-dangle": ["error", "never"],
-        "no-console": 0
+        "no-console": 0,
+        "no-plusplus": "off"
```

Comment:
Is there a reason why this is added now?

Reply:
I'm adding it now because I didn't realise it was a rule until now...and it seems like a really silly rule. There's no need to change it on the mobile app unless you also feel that it should be allowed.

The alternative is replacing everything `n++` or `n--` with `n+=1` or `n-=1`, which I feel is less readable. Also, I have no idea if there's a linting rule that forbids that.

Reply:
Yes that makes sense and I agree with you.